src/components/App/App.css

```diff
@@ -1,124 +1,146 @@
+  height: 20px;
+  position: absolute;
+  border-radius: 50%;
+  left: 15%;
+  transform-origin: 50%;
+  background: radial-gradient(at top left, #34e89e, #0f3443);
+  animation: circle 0.5s alternate infinite ease;
+}
 
+@keyframes circle {
+  0% {
+    top: 60px;
+    height: 5px;
+    border-radius: 50px 50px 25px 25px;
+    transform: scaleX(1.7);
+  }
+  40% {
+    height: 20px;
+    border-radius: 50%;
+    transform: scaleX(1);
+  }
+  100% {
+    top: 0%;
+  }
+}
+.circle:nth-child(2) {
+  left: 45%;
+  animation-delay: 0.2s;
+}
+.circle:nth-child(3) {
+  left: auto;
+  right: 15%;
+  animation-delay: 0.3s;
+}
+.shadow {
+  width: 20px;
+  height: 4px;
+  border-radius: 50%;
+  background-color: rgba(0, 0, 0, 0.5);
+  position: absolute;
+  top: 62px;
+  transform-origin: 50%;
+  z-index: -1;
+  left: 15%;
+  filter: blur(1px);
+  animation: shadow 0.5s alternate infinite ease;
+}
 
+@keyframes shadow {
+  0% {
+    transform: scaleX(1.5);
+  }
+  40% {
+    transform: scaleX(1);
+    opacity: 0.7;
+  }
+  100% {
+    transform: scaleX(0.2);
+    opacity: 0.4;
+  }
+}
+.shadow:nth-child(4) {
+  left: 45%;
+  animation-delay: 0.2s;
+}
+.shadow:nth-child(5) {
+  left: auto;
+  right: 15%;
+  animation-delay: 0.3s;
+}
 
+body {
+  overflow: overlay;
+}
 
-
-/*--------------- Loader css ---------------------*/
-.wrapper{
-    width:200px;
-    height:60px;
-    position: absolute;
-    left:50%;
-    top:50%;
-    transform: translate(-50%, -50%);
-}
-.circle{
-    width:20px;
-    height:20px;
-    position: absolute;
-    border-radius: 50%;
-    left:15%;
-    transform-origin: 50%;
-    background: radial-gradient(at top left, #34e89e, #0f3443);
-    animation: circle .5s alternate infinite ease;
+::-webkit-scrollbar {
+  width: 10px;
 }
 
-@keyframes circle{
-    0%{
-        top:60px;
-        height:5px;
-        border-radius: 50px 50px 25px 25px;
-        transform: scaleX(1.7);
-    }
-    40%{
-        height:20px;
-        border-radius: 50%;
-        transform: scaleX(1);
-    }
-    100%{
-        top:0%;
-    }
-}
-.circle:nth-child(2){
-    left:45%;
-    animation-delay: .2s;
-}
-.circle:nth-child(3){
-    left:auto;
-    right:15%;
-    animation-delay: .3s;
-}
-.shadow{
-    width:20px;
-    height:4px;
-    border-radius: 50%;
-    background-color: rgba(0,0,0,.5);
-    position: absolute;
-    top:62px;
-    transform-origin: 50%;
-    z-index: -1;
-    left:15%;
-    filter: blur(1px);
-    animation: shadow .5s alternate infinite ease;
+::-webkit-scrollbar-track {
+  background: transparent;
 }
 
-@keyframes shadow{
-    0%{
-        transform: scaleX(1.5);
-    }
-    40%{
-        transform: scaleX(1);
-        opacity: .7;
-    }
-    100%{
-        transform: scaleX(.2);
-        opacity: .4;
-    }
-}
-.shadow:nth-child(4){
-    left: 45%;
-    animation-delay: .2s
-}
-.shadow:nth-child(5){
-    left:auto;
-    right:15%;
-    animation-delay: .3s;
-}
\ No newline at end of file
+::-webkit-scrollbar-thumb {
```

Comment:
Could you please explain what is this thumb and above track do? I am not aware of it.

Reply:
thumb refers to the scrollbar that is visible(in colour that goes up and down) and track refers to the total path that the scrollbar is gonna travel. It thereby creates pseudo element of scrollbar. I hope its clear! @bislara

Reply:
thumb refers to the scrollbar that is visible(in colour that goes up and down) and track refers to the total path that the scrollbar is gonna travel. It thereby creates pseudo element of scrollbar. I hope its clear! @bislara

Reply:
Oh okay. Noicee. 🚀🙌